build/README.md

```diff
@@ -692,3 +692,49 @@ After everything builds successfully, the packaging process kicks off and perfor
      📄 devcontainer.json
      📄 Dockerfile
 ```
+
+## Linux ARM64 Specific Builds
+
+The below provides a mechanism to build and test against targeted platforms (e.g. Linux ARM64). The following example uses the the dotnet container image for reference.
+
+### Build
```

Comment:
Let's assume one has gone through this process and validated a new container image for ARM64. How does one then go about adding this to the official build process?

Reply:
Hi @gfmatthews , good question. AFAIK, devcontainer team is the only capable of adding it to the build process. But it would be good to get confirmation from @Chuxel

Reply:
Note that adding an architecture for an existing image name (e.g. mcr.microsoft.com/vscode/devcontainers/dotnet) reuses the same image name, so only the architecture property in definition-manifest.json is needed.